SelectSort.java

```diff
@@ -0,0 +1,30 @@
+public class SelectSort {
+	
+ 	 
+	    public static int[] doSelectionSort(int[] arr){
+	         
+	        for (int i = 0; i < arr.length - 1; i++)
+	        {
+	            int index = i;
+	            for (int j = i + 1; j < arr.length; j++)
+	                if (arr[j] < arr[index]) 
+	                    index = j;
+	      
+	            int smallerNumber = arr[index];  
+	            arr[index] = arr[i];
+	            arr[i] = smallerNumber;
+	        }
+	        return arr;
+	    }
+	     
+	    public static void main(String a[]){
```

Comment:
You can get rid of this useless method.
We use the main method from the App class as a starting point.